src/pages/Career/redux/slice.js

```diff
@@ -0,0 +1,37 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { fetchJobs } from './thunk';
+
+const initialState = {
+  jobs: [],
+  loading: false,
+  error: null,
+};
+
+const careerSlice = createSlice({
+  name: 'career',
+  initialState,
+//   reducers: {
```

Comment:
If you are not using the commented code, you can remove it. 

Reply:
done